src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/ChangeLog.md

```diff
@@ -18,11 +18,10 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Updated all help files to include full parameter types and correct input/output types.
 * Added policy filter to Get-AzureRmRecoveryServicesBackItem cmdlet. The command returns the list of backup items protected by the given policy id.
 * Updated Microsoft.Azure.Management.RecoveryServices.Backup to version 3.0.0-preview.
-
-## Version 4.3.1
```

Comment:
Don't remove or edit previous changelog entries - just add to the changelog entry for the current release, please.